### PR TITLE
SB/24-View-User-Profile-Details

### PR DESCRIPTION
**Pull request by shanedbutler**
# Description

This PR adds a user profile details page navigable from the user list.

# Testing Instructions

1. Launch application from SB/24-View-User-Profile-Details branch
2. In the app, login as admin@example.com
3. Navigate to "Users" on the nav bar
4. From the list of Users click on the eyeball icon
5. You should be redirected to the user details page displaying: 
    > * Full name
    > * Avatar image (if exists, else use a default image)
    > * Display name
    > * Email
    > * Creation Date (MM/DD/YYYY)
    > * User Profile type

If you don't have data for the UserProfile table, you may utilize this SQL query to seed 3 new users to table.
```SQL
INSERT INTO [UserProfile] 
([FirstName], [LastName], [DisplayName], [Email], [CreateDateTime], [ImageLocation], [UserTypeId])
VALUES 
('Jess', 'Ximenez', 'ximjess', 'jess8000@yahoo.com', SYSDATETIME(), 'https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png', 2),
('Rick', 'Rust', 'rustyrick', 'rstnvrslps@hotmail.com', SYSDATETIME(), 'https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png', 2),
('Bianca', 'Bolander', 'bboland', 'berezepz@gmail.com', SYSDATETIME(), 'https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png', 2);
```

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by sarahebrooks12** [comment]
Testing instructions look good --- get 2 testing approvals before merging 👼

**Review by TabathaSlater — APPROVED** [review]
Looks good and  works well on my end

**Pull request merged**
